packages/api/lib/writeRecords/write-granules.js

```diff
@@ -497,7 +497,7 @@ const writeGranuleFromApi = async (
   },
   knex,
   esClient,
-  snsEventType
+  snsEventType = 'Create'
```

Comment:
Why are we adding a default value? I feel like we want to make sure that people specify the correct value when calling this function so not having a default is preferable, but I could be wrong

Reply:
Yeah I was unsure about adding a default. I thought it was wise so we don't have to update every call to `writeGranuleFromApi` but I can definitely see how that's not super compelling.